langchain/prompts/base.py

```diff
@@ -12,6 +12,8 @@
 from langchain.formatting import formatter
 from langchain.schema import BaseMessage, BaseOutputParser, HumanMessage, PromptValue
 
+ACCEPTABLE_PARTIAL_TYPES = Union[str, Callable[[], str], "BasePromptTemplate"]
```

Comment:
Aliases are typed as CamelCase (https://mypy.readthedocs.io/en/stable/kinds_of_types.html#type-aliases)

```suggestion
PartialTypes = Union[str, Callable[[], str], "BasePromptTemplate"]
```